planner_type: global
goal_distance: 2.5
min_similarity: 0.5
max_keep: 8
global_stages: [] # [0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15]
switch_distance: 2.0 # costmap to graph only
